definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,58 @@
+## Various latitude/longitude properties
+
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
+
+These values are machine-generated math centroids for a record's geometry.
+
+### intersection
+
+Find out more about the `intersection` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/intersection).
+
+The `intersection:latitude` and `intersection:longitude` represent the intersection location for a given record.
+
+### lbl
+
+Find out more about the `lbl` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/lbl).
+
+The `lbl:latitude` and `lbl:longitude` represent the label location for a given record.
+
+### local
+
+Find out more about the `local` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/local).
+
+The `local:latitude` and `local:longitude` represent what the "local" centroid for a "place" is. 
```

Comment:
The `local:latitude` and `local:longitude` represent where a local resident would say the center of a place is. For a neighbourhood this could be a subway stop, large park, major street intersection, etc.

`for a "place" is` should be record to be consistent, or take the quote marks off.

Reply:
Take the [Mission](https://whosonfirst.mapzen.com/spelunker/id/1108830809/) in [San Francisco](https://whosonfirst.mapzen.com/spelunker/id/85922583/), for example; locals in San Francisco would likely say the center of the "Mission" is the 16th and Mission St BART Station.